test-integration/operator-tests/src/test/resources/infinispans/client_validation.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: infinispan.org/v1
+kind: Infinispan
+metadata:
+  name: client-validation
+  annotations:
+    infinispan.org/monitoring: 'false'
+spec:
+  expose:
+    type: Route
+  replicas: 1
+  security:
+    endpointAuthentication: true
+    endpointEncryption:
+      type: Service
+      clientCert: Validate
+      clientCertSecretName: client-validation-client-cert-secret
+    endpointSecretName: connect-secret
+  logging:
```

Comment:
Debug can be added should we start encountering issues. Can be removed

Reply:
Removed